packages/circuit-ui/components/BaseStyles/__snapshots__/BaseStylesService.spec.ts.snap

```diff
@@ -175,10 +175,7 @@ exports[`BaseStylesService should return the global base styles 1`] = `
   body {
     background-color: #FFF;
     color: #1A1A1A;
-    
-      font-size: 16px;
-      line-height: 24px;
-    ;;
+    font-size:16px;line-height:24px;;;
```

Comment:
👀 where are these extra `;` coming from

I can see they were already there in the old snapshot so I don't think they're harmful, just strange

Reply:
I don't know, I was wondering the same. I think it's because of how we're accessing the styles in the tests, this doesn't show up when used with a styled component. So, nothing to worry about.